OlinkAnalyze/vignettes/bridging_E3072toEHT.Rmd

```diff
@@ -0,0 +1,682 @@
+  kbl(booktabs = TRUE,
+      digits = 2,
+      caption = "Table 1. Recommended number of bridging samples for Olink platforms") |>
+  kable_styling(bootstrap_options = "striped",
+                full_width = FALSE,
+                position = "center",
+                latex_options = "HOLD_position")
+
+```
+
+
+
+## Workflow Overview
+
+Olink Explore 3072 to Olink Explore HT bridging requires Explore 3072
+data and Explore HT data which have at least 40 to 64 bridging samples. For
+studies containing multiple projects of Explore 3072 data, the Explore
+3072 data sets should be bridged using within-product bridging as
+detailed in the [Introduction to bridging
+tutorial](https://cran.r-project.org/web/packages/OlinkAnalyze/vignettes/bridging_introduction.html) or otherwise normalized together
+prior to performing between-product bridging.
+
+The assays from Explore 3072 are matched to the corresponding assays in
+Explore HT and evaluated to determine if the assay is bridgeable. Additionally, all assays are normalized using both quantile smoothing and
+normalization using the median of paired differences. The result is an
+adjusted Explore 3072 data set with three additional columns:
+
+-   `BridgingRecommendation`: a flag which indicates if the assay is
+    bridgeable and, if so, which normalization method is recommended
+
+-   `MedianCenteredNPX`: NPX values after normalization using the median
+    of paired differences
+
+-   `QSNormalizedNPX`: NPX values after normalization using quantile
+    smoothing
+    
+Note that regardless of the bridging recommendation, NPX values will be
+available for both normalization methods. A visual representation of the between-product bridging workflow is shown below.
+
+```{r, fig.cap= fcap, eval = TRUE, echo = FALSE, out.width="100%"}
+knitr::include_graphics(normalizePath("../man/figures/Bridging_schematic.png"), error = FALSE)
+fcap <- "Schematic of Explore 3072 to Explore HT Bridging Workflow"
+```
+
+## Import NPX files
+
+To normalize Explore 3072 data to Explore HT data, first the two data
+sets are read into R using `read_NPX()`. If more than two data sets are
+being normalized, all Explore 3072 studies should be normalized together
+prior to normalizing between products and the concatenated bridged data
+set should be used as the input. In the case of multiple Explore HT
+studies, only one Explore HT study should be chosen as the reference
+data set. The data can be loaded using `read_NPX()` function with
+default Olink Software NPX file as input, as shown below.
+
+```{r message=FALSE, eval=FALSE, echo = TRUE}
+data_E3072 <- read_NPX("~/NPX_Explore3072_location.parquet") # Could also be a CSV file
+data_EHT <- read_NPX("~/NPX_ExploreHT_location.parquet")
+```
+
+## Checking input datasets and bridging samples
+
+First, confirm that there are overlapping sample IDs within the study.
+In a case where the same sample have different sample IDs in the two projects, the `overlap_sample_list`
```

Comment:
As per 9-SEP-2024 discussion, we may need to remove the line of text relating to differing overlapping sample IDs depending on what we can incorporate into the initial release. TBD on whether we will keep or remove this - for now, is OK to keep.